salt/states/event.py

```diff
@@ -11,19 +11,24 @@
 
 
 def send(name,
-        data=None,
-        preload=None,
-        with_env=False,
-        with_grains=False,
-        with_pillar=False,
-        **kwargs):
+         data=None,
+         preload=None,
+         show_changed=True,
```

Comment:
Are we calling this function positionally anywhere in salt? It'd probably be better to move this to the bottom of the list, just in case?